src/components/pages/create-account/domain/checkIfEmailIsValid.js

```diff
@@ -0,0 +1,5 @@
+export const checkIfEmailIsValid = emailValue => {
+  const emailValidationRegExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
Sur https://emailregex.com/, j'ai trouvé une autre regex (qui est censé gérer aussi les emails exotiques de type `"adrien@localhost"@saunier.fr`).
De manière plus globale

```suggestion
  const emailValidationRegExp = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```